src/photos.cs

```diff
@@ -190,16 +200,15 @@ interface PHAssetChangeRequest {
 
 	[iOS (9,0)]
 	[TV (10,0)]
-	[NoMac]
-	[BaseType (typeof(PHAssetChangeRequest))]
+	[Mac (10,15, onlyOn64: true)]
+	[BaseType (typeof (PHAssetChangeRequest))]
```

Comment:
Unnecessary whitespace change: 

```suggestion
	[BaseType(typeof (PHAssetChangeRequest))]
```